src/app/event/event-service.js

```diff
@@ -100,7 +100,11 @@ function EventService($http, $q, $rootScope, EventFactory) {
     };
 
     this.isDj = function() {
-        return _broadcastEvent !== undefined && _event !== undefined  && _broadcastEvent._id == _event._id;
+        return _broadcastEvent !== undefined && _event !== undefined && _broadcastEvent._id == _event._id;
+    }
+
+    this.isBroadcastActive = function() {
+        return _broadcastEvent && $account.getUser()  && _broadcastEvent.owner_id == $account.getUser()._id ;
```

Comment:
you don't need the user here, because it is always the user itself. you can not have a broadcast event from someone else. you only need to check if the `_broadcastEvent` is set